src/functions/index.js

```diff
@@ -5,12 +5,15 @@ import { handleSearch } from "./search";
 import {
   addIngredient,
   changeRecipeRating,
+  changeRecipeStarsInComments,
   deleteIngredient,
+  editComment,
   formHandleSubmit,
   handleIngredientChange,
   recipeIngredientsCheck,
 } from "./Recipes";
 import { getUser } from "./User";
+
 export {
   //******* Recipes *******
   //form
```

Comment:
In my opinion, if you need a comment that say `recipes` that means that your code is not self explanatory. 

You can export an object names recipes like this : 

```js
export const recipes = {}
```

Then we can call recipe.method and it will be clear where this function came from (This is just my opinion)